deluge/transfer.py

```diff
@@ -40,20 +43,28 @@ def transfer_message(self, data):
         Transfer the data.
 
         The data will be serialized and compressed before being sent.
-        First a header is sent - containing the length of the compressed payload
-        to come as a signed integer. After the header, the payload is transfered.
+        The format is:
```

Comment:
This information should be in the class docstring

Reply:
Done.